code/game/machinery/computer/communications.dm

```diff
@@ -549,7 +549,6 @@
 	if(new_possible_answers)
 		possible_answers = new_possible_answers
 
-#undef IMPORTANT_ACTION_COOLDOWN
```

Comment:
🤨

Reply:
This is so that the fax machine has the same cooldown as the comms console.

Reply:
Move this to the defines folder

Reply:
done